docs/site_pipeline.rst

```diff
@@ -447,6 +447,19 @@ the preprocessing steps applied to them.
    :module: sotodlib.site_pipeline.preprocess_tod
    :func: get_parser
 
+preprocess-obs
+--------------
+This script is set up to run a preprocessing pipeline using the preprocess
+module. See details in :ref:`See details here<preprocess-module>` for how to
+build an obs preprocessing pipeline.
+
+This module is similar to ``preprocess_tod`` but removes grouping by detset so
```

Comment:
I like this maybe add this sort of statement also up when you introduce tod/obs in the preprocess.rst.